server/backend/database/mongo/client.go

```diff
@@ -703,6 +766,37 @@ func (c *Client) FindDeactivateCandidatesPerProject(
 	return clientInfos, nil
 }
 
+// FindDeactivateCandidates finds the clients that need housekeeping.
+func (c *Client) FindDeactivateCandidates(
```

Comment:
Are we still using this function? I don't see this function reference in DB interface.